docs/admin_case_study.md

```diff
@@ -0,0 +1,371 @@
+# Admin case study
+
+This document provides several case studies of `pg_diffix` configuration. The examples are from real databases that are used for the Diffix bounty programs. The databases can be found at https://db001.gda-score.org/.
+
+## The banking database
+
+The banking database is taken from a dataset originally used in a machine learning competition. The data itself is from a Czech bank, and had been pseudonymized to protect privacy. The database is named `banking0`. It has seven tables: `accounts`, `cards`, `clients`, `disp`, `loans`, `orders`, and `transactions`.
+
+### Database preparation
+
+The banking data contains clients and their banking accounts. In its original form, most tables contained a column `account_id` that identifies the account associated with the corresponding row of data. A table `clients` contains information about each client, and a table `accounts` links clients and accounts. An analyst would have to use `JOIN` to include both client and account information in a query.
+
+Diffix Elm does not support SQL JOIN, so the `banking0` tables at https://db001.gda-score.org are built from joins of `accounts` and `clients` with the other tables (`cards`, `orders`, and so on). In other words, all tables contain both account and client information.
+
+The `banking0` database contains both individual and joint accounts. Data about both clients are therefore included in every row of data. The orders table, for instance, has the following columns:
+
+```
+      Column      |       Type       |
+------------------+------------------+
+ order_id         | integer          |
+ bank_to          | text             |
+ account_to       | text             |
+ amount           | double precision |
+ k_symbol         | text             |
+ account_id       | integer          |
+ acct_district_id | integer          |
+ frequency        | text             |
+ acct_date        | text             |
+ client_id1       | integer          |
+ cli_district_id1 | integer          |
+ disp_type1       | text             |
+ birth_number1    | text             |
+ lastname1        | text             |
+ client_id2       | integer          |
+ cli_district_id2 | integer          |
+ disp_type2       | text             |
+ birth_number2    | text             |
+ lastname2        | text             |
+```
+
+The original table contained only the first six columns (ending with `account_id`). During table preparation, we added columns containing account information (`acct_district_id`, `frequency`, and `acct_date`), and columns containing information about both clients. The values for the second client are `NULL` for individual accounts. (Note that the database from which the `banking0` database is derived is `banking`, which can also be viewed at db001.gda-score.org.)
+
+### Activating pg_diffix
+
+The pg_diffix extension must be activated for every database. After connecting to the `banking0` database, the superuser executes:
```

Comment:
> After connecting to the `banking0` database

This might sound like you need to do it every time you connect. A note that it needs to be done once would clarify.
Maybe add something like:

> After the [database preparation[(#database-preparation), the superuser connects to the `banking0` database and activates the pg_diffix extension by executing: